BuildResidentialHPXML/measure.rb

```diff
@@ -833,9 +833,9 @@ def arguments(model)
 
     arg = OpenStudio::Measure::OSArgument::makeStringArgument('heating_system_heating_capacity', true)
     arg.setDisplayName('Heating System: Heating Capacity')
-    arg.setDescription("The output heating capacity of the heating system. If using '#{Constants.SizingAuto}', the autosizing algorithm will use ACCA Manual S to set the capacity.")
+    arg.setDescription("The output heating capacity of the heating system. If using '#{Constants.Auto}', the autosizing algorithm will use ACCA Manual S to set the capacity.")
     arg.setUnits('Btu/hr')
-    arg.setDefaultValue(Constants.SizingAuto)
```

Comment:
Should `SizingAuto` be removed from constants.rb?

Reply:
The HPXMLtoOS measure uses it.